02-express-tutorial/routes/people.js

```diff
@@ -0,0 +1,17 @@
+const express = require("express");
+const router = express.Router();
+
+//let {people} = require('../data')
+
+const { getPeople, addPerson, updatePeople, deletePerson } = require("../controllers/people");
+
+
+router.get('/', getPeople)
```

Comment:
You forgot to add a method to get a person by ID. Please add a route and a controller method that receives the person ID by parameter and returns this person data if it exists or returns an error if it doesn't exist.